src/proctored-exam-settings/ProctoredExamSettings.test.jsx

```diff
@@ -27,36 +78,51 @@ describe('ProctoredExamSettings', () => {
             proctoring_provider: 'proctortrack',
             proctoring_escalation_email: 'test@example.com',
             create_zendesk_tickets: false,
+            course_start_date: '2013-01-01T00:00:00Z',
           },
           available_proctoring_providers: ['software_secure', 'proctortrack', 'mockproc'],
         },
         catch: () => {},
       }),
     }));
-    auth.getAuthenticatedUser = jest.fn(() => ({ userId: 3 }));
+    const waitUntil = () => component.update();
+    await waitUntil();
+
+    expect(component.find('option[value="mockproc"]').prop('disabled')).toEqual(true);
```

Comment:
This test, in addition to the tests that follow, is not really being tested. I need a way to wait for the state to be set before making an assertion on the component. When the assertion is made here, we would expect that `courseStartDate` would be `'2013-01-01T00:00:00Z'`. Instead it is undefined, because the state is not set. Because of this, options on the field will not be properly marked as disabled.

Reply:
This has been resolved, I had course_start_date in the wrong place